gtsfm/frontend/detector_descriptor/superpoint.py

```diff
@@ -0,0 +1,267 @@
+"""Superpoint detector+descriptor implementation.
+
+The network was proposed in 'SuperPoint: Self-Supervised Interest Point Detection and Description' and is implemented
+by wrapping over author's implementation.
+
+References:
+- https://arxiv.org/abs/1712.07629
+- https://github.com/magicleap/SuperGluePretrainedNetwork
```

Comment:
Thanks for the PR, Ayush.
this looks like the link for SuperGlue, not SuperPoint?

Reply:
This repo has the code and model weights for superpoint too.

Reply:
Got it